src/lib/core/CHIPConfig.h

```diff
@@ -1383,7 +1383,7 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
  * @brief Defines the maximum number of path objects, limits the number of attributes being read or subscribed at the same time.
  */
 #ifndef CHIP_IM_SERVER_MAX_NUM_PATH_GROUPS
-#define CHIP_IM_SERVER_MAX_NUM_PATH_GROUPS 8
+#define CHIP_IM_SERVER_MAX_NUM_PATH_GROUPS 9
```

Comment:
Please hold off merging for now as the spec PR has 2x CR as of today: https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/5031

Reply:
I don't think it's the number that's under discussion, but the wording and placement